### PR TITLE
X.26 lint all

### PR DESCRIPTION
**Pull request by peter-lawrey**

### CODE REVIEW

File: src/main/java/net/openhft/chronicle/wire/BinaryWire.java [hunk 1]
```diff
@@ -3563,7 +3563,7 @@ public Object typePrefixOrObject(Class<?> tClass) {
                 if (Wires.dtoInterface(tClass) && GENERATE_TUPLES) {
                     return Wires.tupleFor(tClass, sb.toString());
                 }
-                if (THROW_CNFRE)
+                if (true)
                     throw e;
                 Jvm.warn().on(getClass(), "Unknown class (" + sb + "), perhaps you need to define an alias", e);
                 return null;
```
Comment (yevgenp) [inline src/main/java/net/openhft/chronicle/wire/BinaryWire.java:3569]: This does not make sense, last 2 lines are unreachable. They should be deleted?

File: src/main/java/net/openhft/chronicle/wire/TextWire.java [hunk 2]
```diff
@@ -1852,7 +1852,7 @@ public Class<?> typePrefix() {
                     return classLookup().forName(stringBuilder);
                 } catch (ClassNotFoundRuntimeException e) {
                     // Note: it's not possible to generate a Tuple without an interface implied.
-                    if (THROW_CNFRE)
+                    if (true)
```
Comment (yevgenp) [inline src/main/java/net/openhft/chronicle/wire/TextWire.java]: Same here

File: src/main/java/net/openhft/chronicle/wire/TextWire.java [hunk 3]
```diff
@@ -1893,7 +1893,7 @@ private Object handleCNFE(Class<?> tClass, ClassNotFoundRuntimeException e, Stri
                     return Wires.tupleFor(null, stringBuilder.toString());
                 }
                 String message = "Unable to load " + stringBuilder + ", is a class alias missing.";
-                if (THROW_CNFRE)
+                if (true)
```
Comment (yevgenp) [inline src/main/java/net/openhft/chronicle/wire/TextWire.java]: And here

File: src/main/java/net/openhft/chronicle/wire/TextWire.java [hunk 4]
```diff
@@ -1910,20 +1910,14 @@ private Object handleCNFE(Class<?> tClass, ClassNotFoundRuntimeException e, Stri
                             : classLookup().forName(className);
 
                 } catch (ClassNotFoundRuntimeException e1) {
-                    if (!THROW_CNFRE) {
-                        Jvm.warn().on(getClass(), "ClassNotFoundException class=" + className);
-                        return Wires.tupleFor(tClass, className);
-                    }
+                    throw e;
```
Comment (yevgenp) [inline src/main/java/net/openhft/chronicle/wire/TextWire.java:1901]: And here.

**Review by yevgenp — APPROVED** [review]

File: src/main/java/net/openhft/chronicle/wire/TextWire.java (hunk 4 above)
Comment (peter-lawrey) [reply src/main/java/net/openhft/chronicle/wire/TextWire.java:1901]: Thank you for picking those up

**Pull request merged**